backend/src/main/java/WebInterfaceImpl.java

```diff
@@ -1,9 +1,56 @@
+import model.Customer;
+import model.DigitalPart;
+import model.Order;
+import model.PhysicalPart;
+import storage.DBInterface;
+import storage.PostgresSQLConnector;
+import storage.repository.GenericRepository;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+import java.util.List;
+
 /**
  * Created by danie on 2017-09-28.
  */
 public class WebInterfaceImpl implements WebInterface {
+    DBInterface dbConnector = new PostgresSQLConnector();
+    GenericRepository<Customer> customerGenericRepository = new GenericRepository<Customer>(Customer.class,dbConnector);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)